### DISCUSSION TITLE
remover a opção que os ticket são encerrados automaticamente apos 24hs

### DISCUSSION THREAD

**Discussion by renan-delgado** [rtenorioh/Press-Ticket#128] [General]
percebi que os tickets dos usuarios são encerrados automaticamente após 24Hs... existe alguma maneira de remover essa função?

**Comment by JoseMariani** [comment]
Revisa las opciones de configuraciones! Saludos...

**Comment by rtenorioh** [comment]
Favor verificar no .env do frontend se a variavel abaixo tem algum valor:

REACT_APP_HOURS_CLOSE_TICKETS_AUTO=

pois esta variavel é responsavel por esta fechando os tickets automaticamente, para que nao ocorra deixe em branco como acima
